script.js

```diff
@@ -0,0 +1,72 @@
+window.onload = () => {
+  let pianoContainer = document.getElementsByClassName("container")[0];
```

Comment:
Poți face constante din variabilele pe care nu le schimbi niciodată.

```suggestion
  const pianoContainer = document.getElementsByClassName("container")[0];
```